tools/DevDiagnostics/DevHome.DevDiagnostics/Views/ClipboardMonitoringWindow.xaml

```diff
@@ -13,9 +13,29 @@
     MaxHeight="80"
     MinHeight="80"
     IsMaximizable="False"
-    TaskBarIcon="Images/dd.ico">
+    TaskBarIcon="Images/dd.ico"
+    Closed="ThemeAwareWindow_Closed">
+
+    <Window.SystemBackdrop>
+        <MicaBackdrop/>
+    </Window.SystemBackdrop>
 
     <Grid>
-        <controls:ClipboardMonitorControl x:Name="ClipboardMonitorPanel" />
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Horizontal" Height="31">
+            <Image Source="/Images/dd.ico" Height="16" Width="16" Margin="8,2,0,0" VerticalAlignment="Center"/>
```

Comment:
What are these changes for? Why are we adding an icon to this clipboard window? I thought our text was already getting crunchy.... does this make it worse?

Reply:
In order to leverage the same theme-awareness, I made this view the same as BarWindow in respect of ExtendsContentIntoTitleBar, hence had to add back the icon/title we already had. I will clean this all up once we move to WAS 1.6 later (#3803 tracking)- there's also some overlap with tool-window consistency (#3805 tracking), and the two should probably be done toghether.